package.json

```diff
@@ -39,10 +39,11 @@
     "reformat-files": "prettier --ignore-path .eslintignore --write",
     "preseed": "npm run check-environment",
     "prestart": "npm run check-environment",
+    "seed:experiences": "node ./bin/fillTestData/Experiences.js",
```

Comment:
Currently, when running `npm seed:experiences` you get an error because the script requires parameters. The same for messages, circles, and users. If we want to be able to run each `npm seed:<collection-name>` separately, we could fix all these scripts together by defining default parameters with yargs or something like this.

Reply:
> we could fix all these scripts together by defining default parameters with yargs or something like this.

Can be a concern for follow-up PR.